vm/rust/src/lib.rs

```diff
@@ -711,11 +711,11 @@ fn get_versioned_constants(version: *const c_char) -> VersionedConstants {
         CONSTANTS.get(&"0.13.1".to_string()).unwrap().to_owned()
     } else if version < StarknetVersion::from_str("0.13.2").unwrap() {
         CONSTANTS.get(&"0.13.1.1".to_string()).unwrap().to_owned()
-    } else if version < StarknetVersion::from_str("0.13.2.1").unwrap() {
-        CONSTANTS.get(&"0.13.2".to_string()).unwrap().to_owned()
     } else if version < StarknetVersion::from_str("0.13.3").unwrap() {
-        CONSTANTS.get(&"0.13.3".to_string()).unwrap().to_owned()
+        CONSTANTS.get(&"0.13.2".to_string()).unwrap().to_owned()
     } else if version < StarknetVersion::from_str("0.13.4").unwrap() {
+        CONSTANTS.get(&"0.13.3".to_string()).unwrap().to_owned()
+    } else if version < StarknetVersion::from_str("0.13.5").unwrap() {
```

Comment:
What do you think of having `<=` comparison so our brains can pick this kind of errors better.

Reply:
I think the intended behaviour is: Use 1.3.4 until next version appears. Let me try with >= to see if it works.

Reply:
I'm wondering if we can use `VersionedConstants` directly from blockifier instead of putting them in our repo and load them ourselves in Rust code. I pushed another commit overriding the original one to use it and it works.